components/network/openssh/network-ssh.p5m

```diff
@@ -65,3 +65,5 @@ license openssh.license license="BSD, BSD-like (gsskex)" \
 
 depend type=conditional fmri=pkg:/x11/session/xauth \
     predicate=pkg:/x11/library/libxau
+
+depend type=optional fmri=pkg:/network/ssh-askpass
```

Comment:
Hopefully the new choice of dependency will be better (conditional to get ssh-askpass "dispatcher" if an x-server is installed here)